tests/installation/partitioning_raid.pm

```diff
@@ -171,28 +175,6 @@ sub run() {
         send_key 'down';           #should select first disk'
         wait_idle 5;
     }
-    elsif (get_var('UEFI')) {
-        send_key 'alt-p';
-        assert_screen 'partitioning-size';
-        send_key 'ctrl-a';
-        type_string "200 MB";
-        send_key 'alt-n';
-        assert_screen 'partition-role';
-        send_key "alt-e";          # EFI boot Part match
-        assert_screen 'partitoion-role-EFI';
-        send_key 'alt-n';
-        assert_screen 'partition-format';
-        send_key 'alt-a';          # Formatting options
-        send_key_until_needlematch 'filesystem-fat', 'down';
-        send_key 'alt-m';          #  Mounting options
-        type_string "/boot/efi";
-        send_key 'alt-f';
-        assert_screen 'custompart-efi-boot';
-        send_key 'alt-s';
-        send_key 'right';
-        send_key 'down';           # should select first disk'
-        wait_idle 5;
-    }
```

Comment:
We need to make sure that once a disk is removed from RAID pool the system is able to boot. The code I removed just creates a partition out of RAID pool, that means zero fail tolerance. Does it make sense more now to you?